Sources_v3/StreamChat/Query/Sorting/UserListSortingKey.swift

```diff
@@ -40,3 +40,15 @@ extension UserListSortingKey {
         .init(key: rawValue, ascending: isAscending)
     }
 }
+
+/// In case elements have same or `nil` target sorting values it's not possible to guarantee that order of elements will be the same
+/// all the time. So we need to additionally provide safe sorting option.
+extension Array where Element == Sorting<UserListSortingKey> {
+    func appendingSafeSortingIfNeeded() -> [Sorting<UserListSortingKey>] {
+        guard !contains(where: { $0.key == .id }), !isEmpty else {
+            return self
+        }
```

Comment:
The same as https://github.com/GetStream/stream-chat-swift/pull/594/files#r525915508